shinysdr/i/network/session_http.py

```diff
@@ -38,23 +38,23 @@ def __init__(self, session, wcommon, read_only_dbs, writable_db):
         SlashedResource.__init__(self)
         
         # UI entry point
-        self.putChild('', ElementRenderingResource(_RadioIndexHtmlElement(wcommon)))
+        self.putChild(b'', ElementRenderingResource(_RadioIndexHtmlElement(wcommon)))
         
         # Exported radio control objects
-        self.putChild(CAP_OBJECT_PATH_ELEMENT, BlockResource(session, wcommon, _not_deletable))
+        self.putChild(CAP_OBJECT_PATH_ELEMENT.encode(), BlockResource(session, wcommon, _not_deletable))
```

Comment:
How about using b'' for the constant value since it's always used in this context? If that doesn't work, I'd rather see .encode('utf-8') here because it's the right thing for all URL components, rather than depending on the default encoding under Python 2.

Reply:
I'm not sure I actually buy that utf-8 is correct; arguably it should be utf-8 + urlencode, but since it's a constant that's pure ASCII it's kind of moot. Done.